pgx/src/bgworkers.rs

```diff
@@ -210,10 +212,125 @@ unsafe extern "C" fn worker_spi_sigterm(_signal_args: i32) {
     pg_sys::SetLatch(pg_sys::MyLatch);
 }
 
+/// Dynamic background worker handle
+pub struct BackgroundWorkerHandle {
+    handle: *mut pg_sys::BackgroundWorkerHandle,
+    notify_pid: pg_sys::pid_t,
+}
+
+/// PID
+pub type Pid = pg_sys::pid_t;
+
+/// Dynamic background worker status
+#[derive(Debug, Clone, Copy)]
+pub enum BackgroundWorkerStatus {
+    Started,
+    NotYetStarted,
+    Stopped,
+    PostmasterDied,
+    Untracked,
+    Unknown,
```

Comment:
`::Unknown` seems to be dead code.

Reply:
I can replace it with `BackgroundWorkerStatus` made non-exhaustive (just in case something new appears). Or I can just remove it. The only question is, what do we do if (for some reason?) BgwHandleStatus will have more variants in the future. I understand the possibility of this is slim.

Reply:
We own this code, so fixing pgx is fine.  If it pushes compilation errors down to pgx users, that's a fine breaking change to ensure they're handling all the proper cases.

A blanket "Unknown", on the other hand, is kinda mean.  Especially since we do know, at least as of how Postgres works today.

Reply:
Agree. I pushed an update removing it.